.github/workflows/build-quick.yml

```
@@ -44,6 +44,14 @@ jobs:
           $NewPath = "${CurrentPath}/ninja_bin"
           Add-Content -Path $env:GITHUB_PATH -Value $NewPath
 
+      - name: Set NDK version
+        if: matrix.os != 'windows-latest'
+        run: echo "ANDROID_NDK_HOME=$ANDROID_NDK_LATEST_HOME" >> $GITHUB_ENV
+
+      - name: Set NDK version
+        if: matrix.os == 'windows-latest'
+        run: Add-Content -Path $env:GITHUB_ENV -Value ANDROID_NDK_HOME=$env:ANDROID_NDK_LATEST_HOME
```

I'm sure this will give us some unexpected interaction with NDK 27 (or 26.2+) at some point but

1- can't think of a better way to pin it while they roll out the image
2- it'll just fail fast in the `panic` version check below with a clean message so it won't be mysterious or anything

So, works for me